fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/rdf/impl/RootRdfContext.java

```diff
@@ -71,6 +74,8 @@ public RootRdfContext(final FedoraResource resource,
 
         final Stream.Builder<Triple> b = Stream.builder();
 
+        b.accept(create(subject(), type.asNode(), createURI(REPOSITORY_NAMESPACE + "RepositoryRoot")));
```

Comment:
@awoods see https://github.com/fcrepo4/ontology/pull/35 as a follow-on to the existing JIRA ticket